packages/test/test-end-to-end-tests/src/test/detachedContainerTests.spec.ts

```diff
@@ -898,4 +898,47 @@ describeNoCompat("Detached Container", (getTestObjectProvider) => {
 			"Stored handle should match!!",
 		);
 	});
+
+	it("Compressors sync after detached container attaches and sends an op", async () => {
+		provider.reset();
```

Comment:
Why add this? The provider is reset after every test anyway.

Reply:
Hmmm I see. The beforeEach block creates a loader which is added to the loader container tracker.

Reply:
Moved test to `idCompressor.spec.ts` and don't have to do this hacky reset anymore.